ios/CustomerioReactnative.swift

```diff
@@ -40,8 +40,8 @@ class CustomerioReactnative: NSObject {
                 config.trackingApiUrl = trackingApiUrl
             }
         }
-        if organizationId != "" {
-            initializeInApp(organizationId: organizationId)
+        if let isEnableInApp = configData["enableInApp"] as? Bool, isEnableInApp {
+            initializeInApp()
         }
```

Comment:
Same comment as in the `.kt` files, I believe us removing this parameter entirely would remove backwards compatibility (BC) and introduce a breaking change. 

We could continue to provide BC with:
```swift
        let enableInApp = (configData["enableInApp"] as? Bool) ?? organizationId != "" ?? false
        if enableInApp {
            initializeInApp()
        }
```

Reply:
Ah, now I see this is handled in `src/CustomerioTracking.tsx`. Perfect place for it. Love the solution.